Src/AutoFixture/Kernel/OmitRecursionGuard.cs

```diff
@@ -1,34 +0,0 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Collections;
-
-namespace Ploeh.AutoFixture.Kernel
-{
-    public class OmitRecursionGuard : RecursionGuard
```

Comment:
Why doesn't `OmitRecursionGuard` compile under .NET Core?

I need help understanding what's wrong with it, as it's not clear to me what it's doing wrong... It looks fairly innocuous to me...